app/assets/stylesheets/master.scss

```diff
@@ -232,6 +233,36 @@ a {
   color: #61babb;
 }
 
+.vs {
+  color: black;
+  text-align: center;
+  margin-left: 20%;
+  margin-right: 20%;
+}
+
+.rotate {
+  transform: rotate(180deg);
+  -ms-transform: rotate(180deg); 
+  -moz-transform: rotate(180deg); 
+  -webkit-transform: rotate(180deg); 
+  -o-transform: rotate(180deg); 
+}
```

Comment:
Is that all it took to rotate the board. Thats way easier than we all thought!

Reply:
It rotates works but I am seeing that it breaks the drag and drop. The pieces move in the opposite direction of the drag.